src/metrics/mongoose_instrument.erl

```diff
@@ -0,0 +1,68 @@
+-module(mongoose_instrument).
+
+-export([set_up/1, set_up/3,
+         tear_down/1, tear_down/2,
+         span/4, span/5,
+         execute/3]).
+
+-ignore_xref([set_up/3, tear_down/2, span/4, span/5, execute/3]).
+
+-type event_name() :: atom().
+-type labels() :: #{host_type => mongooseim:host_type()}. % to be extended
+-type metrics() :: #{atom() => spiral | histogram}. % to be extended
```

Comment:
I think this was discussed in detail at the tech meeting, and there was no decision yet. As long as we keep `spiral` and `counter`, we can differentiate between the two in Exometer. We can move to just `counter`, but it would be a one-way operation, and we would lose the distinction.

For me if we decide to drop this, it should be a separate PR.

Reply:
Ok I forgot about that. Thank you. For me a separate PR is also fine.